builders/xcm/xcm-transactor.md

```diff
@@ -0,0 +1,194 @@
+ - **innerCall** — encoded call data of the call that will be executed in the destination chain. This is wrapped with the `asDerivative` option if transacting through the derivative account
+ - **feeLocation** — a multilocation representing the currency being used to pay for the remote call execution
+ - **feePayer** — the address that will pay for the remote XCM execution in the transact through sovereign extrinsic. The fee is charged in the corresponding [XC-20 token](/builders/xcm/xc20/overview/)
+ - **call** — similar to `innerCall`, but it is not wrapped with the `asDerivative` extrinsic
+ - **originKind** — dispatcher of the remote call in the destination chain. There are [four types of dispatchers](https://github.com/paritytech/polkadot/blob/0a34022e31c85001f871bb4067b7d5f5cab91207/xcm/src/v0/mod.rs#L60){target=_blank} available
+
+The XCM-Transactor pallet provides three read-methods:
+
+ - **indexToAccount(index)** — returns the origin chain account associated with the given derivative index
+ - **transactInfoWithWeightLimit(location)** — returns the transact information for a given multilocation
+ - **palletVersion()** — returns current pallet version from storage
+
+## Building an XCM with the XCM-Transactor Pallet {: #build-xcm-xcmtransactor-pallet}
+
+This guide covers building an XCM message for remote executions using the XCM-Transactor pallet, specifically with the `transactThroughDerivative` function. The steps to use the `transactThroughDerivativeMultilocation` function are identical, but instead of the currency ID, you specify a multilocation for the fee token.
+
+!!! note
+    You need to ensure that the call you are going to execute remotely is allowed in the destination chain!
+
+### Checking Prerequisites {: #xcmtransactor-check-prerequisites}
+
+To be able to send the extrinsics in Polkadot.js Apps, you need to have:
+
+ - An [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts){target=_blank} with [funds](https://docs.moonbeam.network/builders/get-started/networks/moonbase/#get-tokens){target=_blank}
+ - The account from which you are going to send the XCM through the XCM-Transactor pallet must also be registered in a given index to be able to operate through a derivative account of the sovereign account. The registration is done through the root account (SUDO in Moonbase Alpha), so contact us to get it registered
+ - Remote calls through XCM-Transactor require `xcUNIT` tokens to pay for the execution fees, which is the [XC-20](/builders/xcm/xc20/){target=_blank} representation of the Alphanet relay chain token `UNIT`. You can acquire some by swapping for DEV tokens (Moonbase Alpha's native token) on [Moonbeam-Swap](https://moonbeam-swap.netlify.app){target=_blank}, a demo Uniswap-V2 clone on Moonbase Alpha
```

Comment:
You could potentially specify that different chains will need different tokens to pay for execution fees, like xcDOT if done on Moonbeam.

Reply:
Good catch, I've rephrased it a bit to be more clear